icicle/include/icicle/vec_ops.h

```diff
@@ -367,11 +367,16 @@ namespace icicle {
    * @param program a class that describes the functionality to run on each set of entries at the vectors.
    * @param size Size of each arrays. The program will be executed size times
    * @param config Configuration for the operation.
+   * @note Using the CUDA backend you can use config.ext to tell the function where each vector resides (device or
```

Comment:
Using the CUDA backend... -> When using the CUDA backend...
be including-->include
"you should have a key "is_<id>_on_device" for the #id vector in data_vec "  - not clear

Reply:
Change the unclear part, is it clear now?